src/split.jl

```diff
@@ -27,31 +27,11 @@ end
 
 ###### findall ##################
 
-function findall(ta::TimeArray{Bool,1})
-    rownums = round(Int64, zeros(sum(ta.values)))
-    j = 1
-    for i in 1:length(ta)
-        if ta.values[i]
-            rownums[j] = i
-            j+=1
-        end
-    end
-    rownums
-end
- 
+findall(ta::TimeArray{Bool,1}) = find(ta.values)
```

Comment:
The `findall` function is really just a specific method of `Base.find` for TimeArrays. Would it make sense to embrace that and just call it `find`?